test/test_demo.py

```diff
@@ -48,6 +53,8 @@ def test_demo(self):
         self.assertEqual(app.test_client.get('/t2')[1].status, 429)
         self.assertEqual(app.test_client.get('/t3')[1].status, 200)
         self.assertEqual(app.test_client.get('/t3')[1].status, 200)
+        self.assertEqual(app.test_client.get('/t4/one')[1].status, 200)
+        self.assertEqual(app.test_client.get('/t4/two')[1].status, 429)
```

Comment:
I believe /t4/one and /t4/two here are not sharing the same limiter

Reply:
shared_limit with a scope should be used

Reply:
My bad - that test didn't run for me because of the call to `app.run(host="0.0.0.0", port=5000, debug=True)` before the test class. I'm assuming this is just there for demo purposes - but maybe that can be moved out. If it's commented out the test will run just fine.